static/js/containers/App.js

```diff
@@ -292,6 +294,9 @@ class App extends React.Component<AppProps> {
             path={`${match.url}privacy-statement`}
             component={PrivacyPolicyPage}
           />
+          <Route path={match.url} render={() => {
+            return (<WidgetList widgetListId={SETTINGS.widgetListId} />)
+          }}/>
```

Comment:
by putting this here, the widgets will appear on all pages, right?

Reply:
Correct.